hotsos/core/plugins/openstack/common.py

```diff
@@ -20,6 +21,18 @@
     MASAKARI_EXCEPTIONS,
 )
 
+OST_EOL_INFO = {
```

Comment:
If you could add a comment to say where you are getting these dates from that would be helpful for when we need to update them in the future.